packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -176,7 +179,14 @@ const Thumbnail: React.FC<ThumbnailProps> = ({
             {...props}
         >
             {aspectRatio === AspectRatio.original ? (
-                <img ref={focusImageRef} className={`${CLASSNAME}__image`} src={image} alt={alt} loading={loading} />
+                <img
+                    ref={focusImageRef}
+                    className={`${CLASSNAME}__image`}
+                    src={image}
+                    alt={alt}
+                    loading={loading}
+                    {...(imgProps || {})}
+                />
```

Comment:
If we want the `imgProps` to not override the props we define in this component, the property spred should be at the top:
```suggestion
                <img
                    {...(imgProps || {})}
                    ref={focusImageRef}
                    className={`${CLASSNAME}__image`}
                    src={image}
                    alt={alt}
                    loading={loading}
                />
```

Reply:
Same thing for the code below

Reply:
yep, totally, I will change it